__init__.py

```diff
@@ -80,9 +81,10 @@ class LocationNotFoundError(ValueError):
 
 class OWMApi(Api):
     ''' Wrapper that defaults to the Mycroft cloud proxy so user's don't need
-        to get their own OWM API keys '''
+        to get their own OWM API keys.
+        However it also handles a local OWM api_key.'''
```

Comment:
Perhaps could rewrite a little eg
> Wrapper for the Open Weather Maps API. 
> By default the Mycroft cloud proxy will be used so user's don't need an API key.
> If an api_key is provided, it will connect directly to the OWM API instead.

Reply:
Are you changing that or shall I?

Reply:
You go for it, I'd have to mess up your PR with a random commit from me